website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -290,9 +290,9 @@ The examples described in this section demonstrate how to export a service named
     $ kubectl --context $CLUSTER2_CONTEXT apply --filename intention.yml
     ```
 
-1. For the services in `cluster-01` that you want to access `backend`, add the following annotations to the service file. To dial the upstream service from an application, ensure that the requests are sent to the correct DNS name as specified in [Service Virtual IP Lookups](/docs/discovery/dns#service-virtual-ip-lookups). 
+1. For the services in `cluster-01` that you want like to deploy and dial `backend` in `cluster-02`, ensure that you add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods before deploying the workload. To dial the upstream service from an application, ensure that the requests are sent to the correct DNS name as specified in [Service Virtual IP Lookups](/docs/discovery/dns#service-virtual-ip-lookups). Below is an example workload with the annotation highlighted that allows the workload to join the mesh, and config provided to the workload that enables the workload to dial the upstream service using the correct DNS name. 
```

Comment:
```suggestion
1. Add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods before deploying the workload so that the services in `cluster-01` can dial `backend` in `cluster-02`. To dial the upstream service from an application, verify that requests are sent to the correct DNS name as specified in [Service Virtual IP Lookups](/docs/discovery/dns#service-virtual-ip-lookups). In the following example, the annotation that allows the workload to join the mesh and the configuration provided to the workload that enables the workload to dial the upstream service using the correct DNS name is highlighted. 
```
It feels like a lot fo information for one step. Can you extract any of it as summary information and add it to the introduction to the steps? It seems that (as with the other section) that the core of the example is to add the annotation to the pod configurations you want to change the behavior of. Maybe move the explanations of what the annotations do to the intro?